sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -340,6 +340,7 @@ queryOrganization
     : (ORDER BY order+=sortItem (',' order+=sortItem)*)?
       (CLUSTER BY clusterBy+=expression (',' clusterBy+=expression)*)?
       (DISTRIBUTE BY distributeBy+=expression (',' distributeBy+=expression)*)?
+      (RANGE PARTITION BY rangePartitionBy+=expression (',' rangePartitionBy+=expression)*)?
```

Comment:
`RANGE DISTRIBUTE BY` and  `DISTRIBUTE BY` will conflict, so  used `RANGE PARTITION BY` here.